test/test.js

```diff
@@ -48,6 +48,19 @@ $.mockjax( {
 	}
 } );
 
+$.mockjax( {
```

Comment:
It's the same thing, as `mockjax` add the handler to the global scope. So it doesn't matter if it's inside the test or outside.

But using outside has its benefits, maybe someone can use it to in his test.